tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source_test.ts

```diff
@@ -215,7 +289,7 @@ describe('tb_feature_flag_data_source', () => {
         );
         expect(dataSource.getFeatures()).toEqual({
           enabledExperimentalPlugins: ['a'],
-          inColab: false,
+          inColab: true,
```

Comment:
Calling out that this behavior IS changing

Reply:
Yes, that's fine. Thanks.